CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 1.5.0 - Unreleased
```

Comment:
Hm, 1.5 or Unreleased? Using a version there is confusing, that's why we usually avoid it.